Assets/Scripts/GenerateAsteroidField.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class GenerateAsteroidField : MonoBehaviour
+{
+
+    public Transform asteroidPrefab;
+    public int fieldRadius = 100;
+    public int asteroidCount = 100;
+    public float scaleFromMultiplier = 0.5f;
+    public float scaleToMultiplier = 5;
+
+    void Start()
+    {
+
+        for (int i = 0; i < asteroidCount; i++)
+        {
+            Transform temp = Instantiate(asteroidPrefab, this.transform.position + Random.insideUnitSphere * fieldRadius, Random.rotation, this.transform); 
+            temp.localScale = temp.localScale * Random.Range(scaleFromMultiplier, scaleToMultiplier);
+        }
```

Comment:
if there's more asteroids, this can potentially be very calculation heavy, or at least a heavy load on loading the level :D

Also generating in sphere is already great, but we'll likely need to see how to generate different shapes as well